### PR TITLE
Add signature form

### PR DESCRIPTION
**Pull request by craigcook**

### CODE REVIEW

File: css/style.scss [hunk 1]
```diff
@@ -5,15 +5,17 @@
...
+@import '~@mozilla-protocol/core/protocol/css/templates/multi-column';
+@import '~@mozilla-protocol/core/protocol/css/components/footer';
 @import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark';
 @import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark-product-mozilla';
-
-@import "footer";
+@import '~@mozilla-protocol/core/protocol/css/components/forms/button-container';
+@import '~@mozilla-protocol/core/protocol/css/components/forms/choice';
+@import '~@mozilla-protocol/core/protocol/css/components/forms/field';
+@import '~@mozilla-protocol/core/protocol/css/components/forms/form';
```
Comment (robhudson) [inline css/style.scss:18]: What's the `~` for the `@import`s do?

File: templates/base.html [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+{#
+ # This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ #}
 
--->
-{% set page_title %}{% block page_title %}MIECO{% endblock %}{% endset %}
+{% set page_title %}{% block page_title %}Mozilla{% endblock %}{% endset %}
```
Comment (robhudson) [inline templates/base.html:7]: Good catch :)

**Review by robhudson — APPROVED** [review]
👍

File: css/style.scss (hunk 1 above)
Comment (craigcook) [reply css/style.scss:18]: oh I don't think it's actually necessary for @import, the tilde seems to be a thing for @use... I'll remove

**Pull request merged**